src/category_theory/limits/constructions/over/default.lean

```diff
@@ -15,21 +15,33 @@ import category_theory.limits.constructions.equalizers
 Declare instances for limits in the over category: If `C` has finite wide pullbacks, `over B` has
 finite limits, and if `C` has arbitrary wide pullbacks then `over B` has limits.
 -/
-universes v u -- morphism levels before object levels. See note [category_theory universes].
+universes w v u -- morphism levels before object levels. See note [category_theory universes].
 
 open category_theory category_theory.limits
 
-variables {J : Type v} [small_category J]
 variables {C : Type u} [category.{v} C]
 variable {X : C}
 
 namespace category_theory.over
 
 /-- Make sure we can derive pullbacks in `over B`. -/
-example {B : C} [has_pullbacks C] : has_pullbacks (over B) := by apply_instance
```

Comment:
Type class inference can't seem to guess the trick with lifting the `walking_cospan`. The construction of limits (of size `v`) in the over category only applies to connected diagrams, so behind the scene, we need to make sure the ulift of `walking_cospan` is also connected. This could in future be generalised to actually prove the existence of all connected limits on level `0`.